packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -382,6 +382,10 @@ class _AppBarState extends State<AppBar> {
 
     Widget title = widget.title;
     if (title != null) {
+      if (defaultTargetPlatform != TargetPlatform.iOS) {
+        title = new Semantics(routeName: true, child: title);
```

Comment:
If the title widget is a string, it's value will end up as the label of a semantics node, not it's value (which according to the docs of isRouteName is used to announce a route)?